airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_full_refresh.py

```diff
@@ -84,13 +84,12 @@ def assert_two_sequential_reads_produce_same_or_subset_records(
 
     def test_sequential_reads(
         self,
-        inputs: ConnectionTestConfig,
         connector_config: SecretDict,
         configured_catalog: ConfiguredAirbyteCatalog,
+        ignored_fields: Optional[Mapping[str, List[IgnoredFieldsConfiguration]]],
```

Comment:
```suggestion
        ignored_fields: Optional[Mapping[str, List[IgnoredFieldsConfiguration]]] = {},
```
Do we need to add a default either here or in the method? If it's optional I wonder what will happen if its not declared, since we used to set a default [here](https://github.com/airbytehq/airbyte/pull/22996/files?diff=unified&w=0#diff-82c6340d6fd6cc2a860e7a6080d0eabf830e4e4504a3a10b6f5ccf08818be126L93)

Reply:
I assigned default value in the fixture

Reply:
Thanks, I saw that - you were a step ahead of me while I was reviewing 😄